python/examples/ingestion_with_python_config/main.py

```diff
@@ -26,7 +27,7 @@
         raise Exception("Missing 'BASE_URI' environment variable.")
 
     # Load your telemetry config
-    telemetry_config = nostromos_lv_426()
+    telemetry_config, rules = nostromos_lv_426()
```

Comment:
This part feels a tad awkward to me, the fact that we return a config a rules separately and leave it up to the caller to then have to create rules manually.

Similar to how instantiating a telemetry config creates your asset, channels, and ingestion config for you, wondering if we could follow a similar model with the new `RuleService` where all you have to do is pass in a list of rule configs and all the create/update happens automatically for you. Let me know if you want to talk about this more. The UX seems a bit disjoint with the current direction that we're going.

Reply:
Discussed offline -- we have a plan to align this UX that is in work now.